infra/tests/tests_serializers.py

```diff
@@ -0,0 +1,44 @@
+'''Unit tests for auth serializers.'''
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+import infra.serializers as serializers
+import infra.models as models
+
+
+User = get_user_model()
+
+
+class TestUserSerializer(TestCase):
+    '''Unit tests for serializer of User.'''
+    def test_fields_equal(self):
+        '''Serializer should return fields of User correctly.'''
+        user = User()
+        expected_keys = {
+            'id', 'last_login', 'first_name', 'last_name', 'email',
+            'is_active', 'date_joined'}
+
+        keys = set(serializers.UserSerializer(user).data.keys())
+        self.assertEqual(keys, expected_keys)
+
+
+class TestOperationLogSerializer(TestCase):
+    '''Unit tests for serializer of operationlog.'''
+    def test_fields_equal(self):
+        '''Serializer should return fields of operationlog correctly.'''
+        department = models.OperationLog()
```

Comment:
The variable name `department` for `models.OperationLog()` is wrong, `log` would be a great choice.
Do not just `Ctrl+C` and `Ctrl+V`, please make sure that you fully understand the code before you write your code.